midealocal/message.py

```diff
@@ -135,7 +135,7 @@ def __init__(self) -> None:
         self._device_type: DeviceType = DeviceType.X00
         self._message_type: MessageType = MessageType.default
         self._body_type: BodyType = BodyType.X00
-        self._protocol_version: ProtocolVersion = ProtocolVersion.V1
+        self._message_protocol_version: ProtocolVersion = ProtocolVersion.V1
```

Comment:
```suggestion
        self._message_protocol_version: ProtocolVersion = 0
```

Reply:
Only renaming in this PR, logic will be changed in an upcoming one